docs/samples/git-setup/clustertemplates/version_4.Y.Z/kustomization.yaml

```diff
@@ -11,6 +11,7 @@ resources:
 - sno-ran-du/ns.yaml
 - sno-ran-du/pull-secret.yaml
 - sno-ran-du/policytemplates-defaults-v1.yaml
+- sno-ran-du/policytemplates-defaults-v2.yaml
```

Comment:
policyTemplate defaults v2 is being used in template v4. It seems like sno-ran-du-v4-Y-Z-4.yaml and clusterinstance-defaults-v4.yaml are missing in kustomization.yaml too.